src/popup/snooze-content.js

```diff
@@ -126,13 +126,44 @@ const resizeHandler = () => {
   });
 };
 
+const keyboardHandler = (e) => {
+  const selectables = [...document.querySelectorAll('.option')];
+  const element = document.activeElement;
+  const index = selectables.indexOf(element) || 0;
+
+  switch (e.keyCode) {
+  case 40: {
+    const nextElement = selectables[index + 1];
+    if (nextElement) {
+      nextElement.focus();
+    }
+    break;
+  }
+  case 38: {
+    const previousElement = selectables[index - 1];
```

Comment:
Perhaps, if index is already negative, we should move to the bottom element, and have them wrap? What do you think?

Reply:
Done.